File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/UpdateBio.java
package me.grishka.houseclub.api.methods;

import me.grishka.houseclub.api.BaseResponse;
import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class UpdateBio extends ClubhouseAPIRequest<BaseResponse> {
    public UpdateBio(String bio) {
        super("POST", "update_bio", BaseResponse.class);
        requestBody = new Body(bio);
    }

    private static class Body {
        public String bio;

        public Body(String bio) {
            this.bio = bio;
        }
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/UpdateBio.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.BaseResponse
import me.grishka.houseclub.api.ClubhouseAPIRequest

class UpdateBio(bio: String) : ClubhouseAPIRequest<BaseResponse?>("POST", "update_bio", BaseResponse::class.java) {
    private class Body(var bio: String)

    init {
        requestBody = Body(bio)
    }
}